### ISSUE TITLE
Missing h2 for call to action 

### ISSUE THREAD

**Issue by darthvadur** [all-purpose/ap-website-v3#99]
There are H3 for the sub "card" style buttons but no H2 heading. So the H3s show as children of the wrong sections. 

Eg. on team page
<img width="1396" alt="Our_Team___All_Purpose" src="https://user-images.githubusercontent.com/955866/110700499-def2b600-81a4-11eb-9039-e5db5ec0e915.png">
